Gopkg.toml

```
@@ -13,42 +37,24 @@ required = [
     "sigs.k8s.io/controller-runtime/pkg/runtime/signals",
     "sigs.k8s.io/controller-runtime/pkg/source",
     "sigs.k8s.io/testing_frameworks/integration", # for integration testing
-	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1",
-    ]
-
-[[constraint]]
-  name = "github.com/joho/godotenv"
-  version = "1.2.0"
-
-[[constraint]]
-  name = "github.com/Azure/azure-sdk-for-go"
-  version = "24.0.0"
-
-[[constraint]]
-  name = "github.com/Azure/go-autorest"
-  version = "11.2.8"
+    "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1",
+]
 
 [[constraint]]
-  name = "github.com/ghodss/yaml"
-  version = "1.0.0"
+  name = "github.com/golang/mock"
+  branch = "master"
```

```suggestion
  version = "1.2.0"
```
Could we pin to a release instead?